modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-oadp-multiple-bsl.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-dpa-multiple-bsl_{context}"]
+= Configuring the DPA with more than one BSL
+
+You can configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
+
+.Prerequisites
+
+* You must install the {oadp-short} Operator.
+* You must create the secrets by using the credentials provided by the cloud provider.
+
+.Procedure
+
+. Configure the DPA with more than one BSL. See the following example.
++
+.Example DPA
+[source,yaml]
+----
+#...
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields:~:text=Include%20the%20apiVersion%20and%20kind%20so%20that%20a%20user%20always%20knows%20the%20context%20of%20the%20YAML.

Reply:
The YAML blocks here and in other spots are missing some required fields that cannot be omitted, `APIversion` and `kind`

Reply:
Added the `APIversion` and `kind` fields at all the locations where they were missing. Thanks.